docs/sources/operators-guide/architecture/components/ingester.md

```diff
@@ -63,6 +64,14 @@ If an ingester fails, a subsequent process restart replays the WAL and recovers
 Contrary to the sole replication, and given that the persistent disk data is not lost, in the event of the failure of multiple ingesters, each ingester recovers the in-memory series samples from WAL after a subsequent restart.
 Replication is still recommended in order to gracefully handle a single ingester failure.
 
+### Write-behind log
+
+The write-behind log (WBL) is similar to the WAL, but it only writes incoming out-of-order samples to a persistent disk until the series are uploaded to long-term storage.
+
+There is a different log for this because it is not possible to know if a sample is out-of-order until you try to append it. First you need to attempt to append it, the TSDB will detect that it is out-of-order, append it anyway if out-of-order is enabled and then write it to the log.
```

Comment:
I think we use "You" to refer to the user. Instead of "you" or "we", shall we just say "Mimir"?

(Also can we keep use line = one sentence format?)